lib/pages/chat_details/chat_details_view.dart

```diff
@@ -36,46 +38,23 @@ class ChatDetailsView extends StatelessWidget {
         return Scaffold(
           floatingActionButton: _AddMembersButton(controller: controller),
           backgroundColor: LinagoraSysColors.material().onPrimary,
-          appBar: AppBar(
-            backgroundColor: LinagoraSysColors.material().onPrimary,
-            automaticallyImplyLeading: false,
-            bottom: PreferredSize(
-              preferredSize: const Size(double.infinity, 1),
-              child: Container(
-                color: LinagoraStateLayer(
-                  LinagoraSysColors.material().surfaceTint,
-                ).opacityLayer1,
-                height: 1,
-              ),
-            ),
-            title: Padding(
-              padding: ChatDetailViewStyle.navigationAppBarPadding,
-              child: Row(
-                children: [
-                  Padding(
-                    padding: ChatDetailViewStyle.backIconPadding,
-                    child: IconButton(
-                      splashColor: Colors.transparent,
-                      hoverColor: Colors.transparent,
-                      highlightColor: Colors.transparent,
-                      onPressed: controller.widget.closeRightColumn,
-                      icon: controller.widget.isInStack
-                          ? const Icon(
-                              Icons.chevron_left_outlined,
-                            )
-                          : const Icon(Icons.close),
-                    ),
-                  ),
-                  Flexible(
-                    child: Text(
-                      L10n.of(context)!.groupInformation,
-                      style: Theme.of(context).textTheme.titleLarge,
-                      overflow: TextOverflow.ellipsis,
-                    ),
-                  ),
-                ],
-              ),
+          appBar: TwakeAppBar(
+            backgroundColor: controller.responsive.isMobile(context)
+                ? LinagoraSysColors.material().surface
+                : LinagoraSysColors.material().onPrimary,
```

Comment:
why we need to customize it? why not use the default one?

Reply:
some appbars have different color

![image](https://github.com/user-attachments/assets/474cd581-185a-473d-ad54-5015a53c203b)

Reply:
please choose `LinagoraSysColors.material().onPrimary;`

Reply:
for both web and mobile ?

Reply:
remove it